src/compiler/factory.ts

```diff
@@ -214,61 +214,107 @@ namespace ts {
             : node;
     }
 
-    // Type Elements
+    // Signature elements
 
-    export function createSignatureDeclaration(kind: SyntaxKind, typeParameters: TypeParameterDeclaration[] | undefined, parameters: ParameterDeclaration[], type: TypeNode | undefined) {

```

Comment:
Mostly reordered to align with the order in `SyntaxKind`. It makes finding things easier.